README.md

```diff
@@ -211,6 +211,14 @@ the Girder API server running).
 When running against your own instance of the Girder API server,
 make sure to set [CORS](https://girder.readthedocs.io/en/stable/security.html#cors-cross-origin-resource-sharing) accordingly.
 
+### Displaying authenticated media from Girder on a page
+
+If your app injects media dynamically into the page using `img` or `video` elements that load from Girder, there are several requirements.
+
+* You must use the `authenticateWithCredentials` option in the GirderRest constructor.
```

Comment:
```suggestion
* If the data is private, you must use the `authenticateWithCredentials` option in the GirderRest constructor.
```

Reply:
This subsection is called "Displaying authenticated media from Girder on a page".  I believe if the data is public, you don't have to take any of these steps, is that right?

Should I rename the subsection "Displaying private media from Girder on a page" for clarity or "Displaying media from Girder on a page" and explain that, in the public case, everything should just work?

Reply:
> if the data is public, you don't have to take any of these steps, is that right

If the data is public, you still need at least a wildcard ACAO, but you're correct that you don't have to take any of these specific steps, and that's out of scope for this section. Let's go with "displaying private media" and leave the instructions as is.